app/controllers/api/departures_controller.rb

```diff
@@ -32,4 +38,18 @@ def time
     # offset to work correctly I think.
     params[:time] ? Time.zone.parse(params[:time]) : Time.current
   end
+
+  def start_time
+    time - 10.minutes
+  end
+
+  def end_time
+    time + end_time_offset.hour
+  end
+
+  DEFAULT_END_TIME_OFFSET = 1
```

Comment:
Not totally clear from param name or constant that it's expected to be hours (I think)